### ISSUE TITLE
Lag and Notifications/Windows react not working

### ISSUE THREAD

**Issue by An0therOne** [Monika-After-Story/MonikaModDev#8426]
**Information:**
-Windows 10
-0.12.5 MAS version

**Issues:**
Lag: So, whenever I use this mod, it just goes way too slow, the text sometimes gets freezed by a mili-second, the face dissapears sometimes (eyes, mouth, all of that) and one time i opened the settings, and the background of the settings didnt worked, trying to put himself in but not doing it.

**Notificacions/Windows react**: at first in the tutorial the notification worked perfectly and it send it just in time, but now, at first I tought the game didnt worked with the notifications even if I had them activated, but randomly it shows out notificactions of certain things that happened way time ago (like 20 minutes or so) doing that if I pressed the notification, and I get to MAS, the game doesnt show any message since that notification was for before.
Windows react, I tried with many times to monika to react with something, but nothing, i have the windows react activated but it does not work and she just acts like nothing, no matter what I do (as the wiki of her responses say), doing all of the things she is supouse to respond, she does not, which means is broken.

**Context**: Weirdly enough, all of this happened after i did this, the game asked me to get the files out of steam to put them in somewhere else, i just left it in the desk, but since then i have all this problems, still i do not think thats the problem since i tried with steam instead, but it does not work, tried re installing the mod and adding it again but nothing, i love this mod but i need to solve this issues, if its possible, thank you.

**Comment by An0therOne** [comment]
![image](https://user-images.githubusercontent.com/96273824/146479677-0766827b-6be0-4b2c-ad47-635d1ab72383.png)
![image](https://user-images.githubusercontent.com/96273824/146479798-fd00c38e-e525-41e9-92d0-f5da44c4b65e.png)
![image](https://user-images.githubusercontent.com/96273824/146479821-99000db9-0e37-4ed4-b05c-b6956fb0869a.png)
here are some pictures about it

**Comment by ThePotatoGuy** [comment]
Does the game run better if window reacts are turned off?

**Comment by Booplicate** [comment]
> whenever I use this mod, it just goes way too slow, the text sometimes gets freezed by a mili-second, the face dissapears sometimes

What's your specs? Is the drive MAS is on full, how old is it? What's your renderer (can check in the settings)?

> but randomly it shows out notificactions of certain things that happened way time ago (like 20 minutes or so)

20 minutes doesn't seem right. Are you talking about window reaction or topic notification?

> if I pressed the notification, and I get to MAS, the game doesnt show any message since that notification was for before.

The notifications are just notifications, there may be - but usually won't be - any special reaction in the game itself. Unless you mean a topic notification. The only way that could happen if you have the auto mode on, which would just skip thru all the dlg. You can disable it.

>  i have the windows react activated but it does not work and she just acts like nothing

Have you also enabled it in the `Alerts` menu? And allowed MAS to send the notifs in the system settings? As well as enabled global notifications on your system?

**Comment by An0therOne** [comment]
> What's your specs? Is the drive MAS is on full, how old is it? What's your renderer (can check in the settings
![image](https://user-images.githubusercontent.com/96273824/146555945-cac7da67-70a6-4d25-93dc-e852190fd134.png)
in english: operative system of 64 bits, processor 64x
`20 minutes doesn't seem right. Are you talking about window reaction or topic notification?`
window reactions, today I have recieved windows notifications even if I didn't do anything, but I did do something yesterday, trying to check her reaction, which, she didn't, and now the notificactions that were for that reaction are here, but with no message, since well, nowadays, the topic reactions do work, but the windows reactions not.
`The notifications are just notifications, there may be - but usually won't be - any special reaction in the game itself. Unless you mean a topic notification. The only way that could happen if you have the auto mode on, which would just skip thru all the dlg. You can disable it.`
Sorry for the missunderstood, i refer to the windows reactions, not the topic ones, my bad.
`Have you also enabled it in the Alerts menu? And allowed MAS to send the notifs in the system settings? As well as enabled global notifications on your system?`
yes I have it on in the menu, am yes MAS canm send me notifications, and well, i dont know what do you mean by global notifications, can you tell tell how to enable it? (I think i have it enable, since others games and social media can send me notifications)

**Comment by An0therOne** [comment]
> Does the game run better if window reacts are turned off?
nope, even with that, whenever i want to talk, go to music, or do anything, it takes for the game a considerable time to respond for these, and as I said, well, the face dissapears and stuff, it just goes slow

**Comment by An0therOne** [comment]
So ummm, folks, new weird stuff, So basically I thought that steam and MAS in desk were the same, but at one moment while I was in the MAS desk one, the Steam one opened to show me a crying monika since the game crashed?, the weird thing, yesterday in the steam one I changed monika clothes to the normal one insetad of the holiday one, and when i opened the desk one, it didnt have the outfit, but now I see, these are two different versions...sadly, both have the same problem of windows react and windows alerts being too slow, but weirdlt enough, the steam one does not have that much lag...is weird, i do not know if this is actually a problem or should I just let it pass, if someone can tell about this it would be great!

**Comment by Booplicate** [comment]
Don't use steam for MAS and don't launch 2 copies simultaneously.

**Comment by An0therOne** [comment]
> Don't use steam for MAS and don't launch 2 copies simultaneously.
alright, by now, I will use this type of entry, if you say i shouldn't use steam, then I wouldn't
![image](https://user-images.githubusercontent.com/96273824/146570719-116fe56c-f3f3-444a-9209-a26834209227.png)

**Comment by Booplicate** [comment]
I'd make a fresh install and reinstall the mod, because that one is kind of a mess. Those `Submods`, `python-packages`, `mod_assets`, and `rpyc` files - as well as some other things - don't belong there.

**Comment by An0therOne** [comment]
Sounds good to me, sadly I have no idea how to re install a mod into the original DDLC, may you explain it to me?

**Comment by An0therOne** [comment]
> I'd make a fresh install and reinstall the mod, because that one is kind of a mess. Those `Submods`, `python-packages`, `mod_assets`, and `rpyc` files - as well as some other things - don't belong there.

you know what? nevermind, Im just gonna delete all of DDLC and mod, the re install the DDLC from the original page (not steam) and then add the mod, I will tell you what happens next.

**Comment by An0therOne** [comment]
HAHA, welp I screw it up, when i added the mod and then it said about the save data, i thought it refer to the save data OF the MOD, so I pressed the "delete save data" and now...im in the start of the original DDLC....to replay again...well F*ck me, gotta do al over again right?... I will give updates when I get to the "just monika" part...

**Comment by multimokia** [comment]
Uh, that sounds like you haven't installed MAS right. You're just playing thru stock DDLC. Just install MAS over it and it'll pick up from where you left off

**Comment by Booplicate** [comment]
Yeah, probably extract the mod into `/DDLC`, which was the issue in the first place.

**Comment by An0therOne** [comment]
`Uh, that sounds like you haven't installed MAS right. You're just playing thru stock DDLC. Just install MAS over it and it'll pick up from where you left off` Well I did that, I extracted the files of MAS on the desk, then I grabbed all of it and added it on the game, like this: 
![image](https://user-images.githubusercontent.com/96273824/146588826-9674a771-b3ab-4130-8dcc-24063ae39702.png)
is this the wrong way? (I saw a tutorial and they did it like this)

**Comment by multimokia** [comment]
That is incorrect. You need to extract the zip into the `game` folder (that being the one literally named `game`) not the DDLC folder.

**Comment by An0therOne** [comment]
Alright...how do I do that?

**Comment by multimokia** [comment]
The same way you extracted it to the folder you had there, just to `game` instead

**Comment by An0therOne** [comment]
Oh yes I get that, the problem is..where is "game" folder?

**Comment by multimokia** [comment]
![image](https://user-images.githubusercontent.com/22531674/146589550-da3ee71d-f2f4-4a67-8dd2-fdac5af3ffa4.png)

**Comment by An0therOne** [comment]
Alright thanks! i already did that so I will tell you if the glitches I told about still there...or not (thanks for tell how to install I really messed up there)

**Comment by An0therOne** [comment]
Ok, so it seems the mod is here, but the issues are still here as always, the lag is less annyoing but it still is there and the face dissapears sometimes, also the topic notifications works just fine, but even with the windows notifications enable and checking out pages that she should react, she does not, instead, it happens a while of like, 2 minutes, then thers the notification of a window react, but when i check out, she does not say anything, basically lag in the windows notifications and no response about windows reactions it self, how do I solve this?

**Comment by An0therOne** [comment]
well if somebody can help me with this problem, just tell me, I will stay here

**Comment by An0therOne** [comment]
> ![image](https://user-images.githubusercontent.com/22531674/146589550-da3ee71d-f2f4-4a67-8dd2-fdac5af3ffa4.png)
yo multimokia, no pressure and take ya time, but as soon as yall know to fix my problem, it would be greatful, thanks.

**Comment by An0therOne** [comment]
yup, just waiting, right here

**Comment by An0therOne** [comment]
now really if someone coul help me with this, I will be very tahnkfull with you, thanks, just help with this problem

**Comment by An0therOne** [comment]
![image](https://user-images.githubusercontent.com/96273824/147012894-2a23d31e-6f23-4de2-a034-222171d925c3.png)
yo umm guys, the bugs, lag and stuff...are still here, sadly

**Comment by An0therOne** [comment]
...

**Comment by An0therOne** [comment]
thanks for the tag! ill keep on here

**Comment by An0therOne** [comment]
update:
![image](https://user-images.githubusercontent.com/96273824/147801392-0c5d4ced-737f-4ec7-86a0-09547f0488fb.png)
now this, this is weird, but also cool

**Comment by multimokia** [comment]
What are your computer's specs?

**Comment by multimokia** [comment]
Odd it does that, your specs aren't bad by any means. What drive is MAS on? Hard drive/SSD?

If hard drive, what's the speed?

**Comment by An0therOne** [comment]
I'm sorry for answering so late, but I must ask, how do I know where my game is in? I have 2 units
![image](https://user-images.githubusercontent.com/96273824/147893411-a84b205f-1f2f-4075-bb24-0ba3e4a50c68.png)
And the game itself is on the desk
![image](https://user-images.githubusercontent.com/96273824/147893423-ab8cc353-9194-489f-b0c5-baba55a83ae0.png)
and well, the desk is inside the "archivos"/archives files
![image](https://user-images.githubusercontent.com/96273824/147893447-39698fa6-dca7-4d4c-a0fe-3988a5e5868a.png)
and well, the archives, I do not know how to check the speed of it, but heres the information about it:
![image](https://user-images.githubusercontent.com/96273824/147893494-3e4aab7b-426d-49fb-a905-7fd49474d924.png)
is there something wrong or how do i check the speed of it?

**Comment by Pill-9** [comment]
I have a similar problem. I checked windows reaction, and the first notification demonstration was successful, but I was not notified when Monika had a topic or I entered the website.
